packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-right-demo/topic-teaser-right-demo.component.html

```diff
@@ -17,8 +17,8 @@ <h2 class="topic-teaser-title font-curve-large mb-large">
           </ul>
         </div>
 
-        <div class="topic-teaser-image-container col-10 col-rg-8 col-lg-4">
-          <img class="topic-teaser-image" src="assets/images/gazelle.jpg" alt="Gazelle" />
```

Comment:
If these images are no longer used, we should delete them from the assets and the [attribution file](https://github.com/swisspost/common-web-frontend/blob/main/packages/web-demo/src/assets/images/ATTRIBUTION.md).

Reply:
done